### PR TITLE
Python code injection rules

### PR DESCRIPTION
**Pull request by inkz**
More command injection related Python rules

### CODE REVIEW

File: python/lang/security/audit/non-literal-import.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+rules:
+- id: non-literal-import
+  patterns:
+  - pattern: |
+      importlib.import_module($NAME, ...)
+  - pattern-not: |
+      importlib.import_module("...", ...)
+  message: Untrusted user input in `importlib.import_module()` function allows an attacker to load arbitrary code.
```
Comment (minusworld) [inline python/lang/security/audit/non-literal-import.yaml]: What's the proper way to fix this? Is it just to use the regular import syntax?

File: python/lang/security/deserialization/avoid-jsonpickle.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    owasp: 'A8: Insecure Deserialization'
+    cwe: 'CWE-502: Deserialization of Untrusted Data'
+    references:
+    - https://github.com/jsonpickle/jsonpickle#jsonpickle
+    - https://www.exploit-db.com/exploits/49585
+    category: security
+  message: |
+    Avoid using `jsonpickle`, which is known to lead to code execution vulnerabilities.
+    When unpickling, the serialized data could be manipulated to run arbitrary code.
+    Instead, consider serializing the relevant data as JSON or a similar text-based
```
Comment (minusworld) [inline python/lang/security/deserialization/avoid-jsonpickle.yaml]: Recommend changing this to say using the `json` module instead

File: python/lang/security/audit/non-literal-import.yaml (hunk 1 above)
Comment (inkz) [reply python/lang/security/audit/non-literal-import.yaml]: Updated:+1:

File: python/lang/security/deserialization/avoid-jsonpickle.yaml (hunk 2 above)
Comment (inkz) [reply python/lang/security/deserialization/avoid-jsonpickle.yaml]: Updated:+1:

**Review by minusworld — APPROVED** [review]

**Pull request merged**